File: src/Java/FizzBuzz/FizzBuzz.java
public class FizzBuzz {
    public static void main(String args[]) {
        Kattio io = new Kattio(System.in);
        int x = io.getInt(), y = io.getInt(), n = io.getInt();
        for (int i = 1; i < n + 1; i++) {
            if (i % x != 0 && i % y != 0) {
                io.println(i);
            } else if (i % x == 0 && i % y != 0) {
                io.println("Fizz");
            } else if (i % x != 0 && i % y == 0) {
                io.println("Buzz");
            } else {
                io.println("FizzBuzz");
            }
        }
        io.flush();
    }
}

File: src/Python 3/FizzBuzz/FizzBuzz.py
def isDivisible(n, a):
    if n % a == 0:
        return True
    else:
        return False
        
x, y, n = input().split()

x = int(x)
y = int(y)
n = int(n)

for _ in range(1, n + 1):
    if isDivisible(_, x) and isDivisible(_, y):
        print('FizzBuzz')
    elif isDivisible(_, x):
        print('Fizz')
    elif isDivisible(_, y):
        print('Buzz')
    else:
        print(_)